test/bats/tests/config/dynamic_plugins_config.json

```diff
@@ -0,0 +1,57 @@
+{
+    "store": {
+        "version": "1.0.0",
+        "plugins": [
+            {
+                "name": "oras",
+                "cosignEnabled": true
+            }
+        ]
+    },
+    "policy": {
+        "version": "1.0.0",
+        "plugin": {
+            "name": "configPolicy"
+        }
+    },
+    "verifier": {
+        "version": "1.0.0",
+        "plugins": [
+            {
+                "name": "dynamic",
+                "artifactTypes": "sbom/example",
+                "nestedReferences": "application/vnd.cncf.notary.signature",
+                "source": {
+                    "artifact": "index.docker.io/acanthamoeba/sample-plugin:v1"
```

Comment:
TODO: replace with `wabbitnetworks.azurecr.io/test/sample-verifier-plugin:v1` after maintainers push a sample plugin

Reply:
we have a tracking [issue](https://github.com/deislabs/ratify/issues/521)  to discuss if we should continue using wabbitnetworks.  I think the long term goal is to build images for testing locally. Let us know what you think, thank Noel!

Reply:
No strong opinion from my side, at least for this PR. It would make test setup a bit more self contained at the cost of increased build times on all commits. Whatever we choose for the `oras` store should work here as well